app/views/shared/_sponsors.html.haml

```diff
@@ -1,4 +1,4 @@
-.row.align-items-center
+.row.d-flex.align-items-center.justify-content-center
   - sponsors.each do |sponsor|
-    .col-4.col-md-3.col-lg-2.mt-4.d-flex.justify-content-center
-      = link_to image_tag(sponsor.avatar, class: 'small-image mw-100', alt: sponsor.name), sponsor.website, title: sponsor.name, class: 'd-inline-block border-0'
+    .col-4.col-md-3.col-lg-2.mt-4
+      = link_to image_tag(sponsor.avatar, alt: sponsor.name), sponsor.website, title: sponsor.name, class: 'border-0 mw-100'
```

Comment:
@KimberleyCook sorry, I just remembered there was a good reason why this image had the `small-image` class. It was to restrict the height of vertical logos, otherwise sometimes they'd get too tall. This class adds a `max-width: 100px` style to logos. I think we are going to be massaging this section for forever, it's really hard to find a style that works for the myriads of logo formats we have but this is needed.

Reply:
Ok gotcha, I'll add it back.